src/assets/scss/components/Notifications.scss

```
@@ -0,0 +1,11 @@
+button.pf-c-button.pf-m-plain.times-logo {
+    padding-right: 0.5em;
+}
+
+button.pf-c-button.pf-m-plain.notification-close {
+    padding-top: 0em;
```

`em` unit is redundant
```suggestion
    padding-top: 0;
```